packages/kbn-config/src/env.ts

```diff
@@ -22,7 +22,7 @@ import { PackageInfo, EnvironmentMode } from './types';
 
 // `require` is necessary for this to work inside x-pack code as well
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const pkg = require('../../../../package.json');
+const pkg = require('../../../package.json');
```

Comment:
Now that this is in a package, these hardcoded requires won't work anymore, as the compiled file in `node_modules` it not at the same place, or depth, than in the source. I think we don't have any other option than using the `REPO_ROOT` constant from `@kbn/dev-utils` (same lower in file)

Note that as `REPO_ROOT` is an absolute path, we won't be able to `require` it and we'll have to fallback to read and parse the file 'manually'.

The mocking (see `packages/kbn-config/src/env.test.mocks.ts`) will have to be adapted too.

Reply:
Commented on this here: https://github.com/elastic/kibana/pull/76874#discussion_r485173299

Reply:
This is almost ready for review and I have resolved the tests mocking fs by being explicit about the actual files they wish to mock - https://github.com/elastic/kibana/pull/76518